examples/btc-example/src/app/btc.virtualAccount.example.ts

```diff
@@ -0,0 +1,65 @@
+import { TatumBtcSDK } from '@tatumio/btc'
+import { REPLACE_ME_WITH_TATUM_API_KEY } from '@tatumio/shared-testing-common'
+
+export async function btcVirtualAccountExample() {
+  // Virtual account example
+  // We will receive assets on account and withdraw it
+  // More info here: https://docs.tatum.io/guides/ledger-and-off-chain
+
+  const btcSDK = TatumBtcSDK({ apiKey: REPLACE_ME_WITH_TATUM_API_KEY })
+
+  // Check whether a blockchain address is assigned to a virtual account
+  // You can find more details in https://apidoc.tatum.io/tag/Blockchain-addresses#operation/addressExists
+  const account = await btcSDK.virtualAccount.depositAddress.checkExists('1N4U6RidG5XScvBoSNgq5EmHiPxU4MAyEv')
+  console.log(account)
+
+  // Create a deposit address for a virtual account
+  // You can find more details in https://apidoc.tatum.io/tag/Blockchain-addresses#operation/generateDepositAddress
+  const address = await btcSDK.virtualAccount.depositAddress.create('5e68c66581f2ee32bc354087', 1)
```

Comment:
you must first create wallet, xpub, virtual account and then generate deposit address - full flow must be there